packages/handlersjs-logging/lib/loggers/pino-logger.ts

```diff
@@ -39,69 +71,41 @@ export class PinoLogger extends Logger {
 
     if (level <= this.minimumLevel) {
 
-      const logData = level > this.minimumLevelPrintData
+      const parsedData = level > this.minimumLevelPrintData
         ? {}
         // if data is a string, convert it to an object with a data property. Pino expects data to be an object. If it remains a string, it will use the data as a message, and drop the rest of our log.
         : typeof data === 'string'
-          ? { data }
+          ? { string: data }
           // if data is undefined, convert it to an empty object.
           : data ?? {};
 
+      const logData = { variables: this.variables, data: parsedData };
```

Comment:
if data is a string, this will result in a data object with a nested "string" key.

For example logger.info('data', 'message') will result in
```
{
  variables: { ... },
  data: {
    string: 'data',
  },
}
```

Don't we just want something like this?:
```
{
  variables: { ... },
  data: 'data',
}
```

Reply:
That would make sense but that is not how it was implemented before.
When passed a string `hello mate`, it would create an object `{ "hello mate": "hello mate" }`.

Reply:
Comment from the line above:
```
// if data is a string, convert it to an object with a data property.
Pino expects data to be an object.
If it remains a string, it will use the data as a message,
and drop the rest of our log.
```

Reply:
Since it will alwasy be an object now data can idd just be a string. Just pushed the change